assets/src/common/helpers/index.js

```diff
@@ -55,11 +55,10 @@ export const hasMinimumDimensions = ( media, dimensions ) => {
  * @return {Object} Minimum dimensions including width and height.
  */
 export const getMinimumFeaturedImageDimensions = () => {
-	const width = MINIMUM_FEATURED_IMAGE_WIDTH;
-
-	const height = width * ( 9 / 16 );
-
-	return { width, height };
+	return {
+		width: parseInt( featuredImageMinimumWidth ),
+		height: parseInt( featuredImageMinimumHeight ),
```

Comment:
The use of `parseInt()` seems obsolete now.

```suggestion
		width: featuredImageMinimumWidth,
		height: featuredImageMinimumHeight,
```

Reply:
@westonruter wp_localize_script will provide it as a string even though we supply integer to it, we still need to convert it right ?

Reply:
Oh yes, you're right. Please then also change this:

https://github.com/ampproject/amp-wp/blob/a1a7e8cb7d085bd06f73fb1d25cd300670dc0c22/includes/admin/class-amp-post-meta-box.php#L275-L279

To:

```php
wp_add_inline_script(
	self::BLOCK_ASSET_HANDLE,
	sprintf( 'var ampBlockEditor = %s;', wp_json_encode( $data ) ),
	'before'
);
```

I consider this to be a best practice for exporting PHP data to JS instead of using `wp_localize_script()`.

Reply:
Done, i will fix the tests.